engine/src/main/java/org/destinationsol/game/DebugOptions.java

```diff
@@ -19,71 +19,69 @@
 import org.destinationsol.IniReader;
 import org.destinationsol.SolFileReader;
 
-public class DebugOptions {
-    public static final Vector2 DEBUG_POINT = new Vector2();
-    public static final Vector2 DEBUG_POINT2 = new Vector2();
-    public static final Vector2 DEBUG_POINT3 = new Vector2();
-    public static final boolean ASSERTIONS = false;
+public final class DebugOptions {
+    public static MissingResourceAction missingPhysicsAction;
+
     public static final boolean PRINT_BALANCE = false;
-    public static String DEV_ROOT_PATH;
+    public static String developmentRootPath;
+    static final boolean ASSERTIONS = false;
+    static final Vector2 DEBUG_POINT = new Vector2();
+    static final Vector2 DEBUG_POINT2 = new Vector2();
+    static final Vector2 DEBUG_POINT3 = new Vector2();
 
     // World initialization
-    public static boolean EMULATE_MOBILE = false;
-    public static String SPAWN_PLACE = "";
-    public static String FORCE_PLANET_TYPE = "";
-    public static String FORCE_SYSTEM_TYPE = "";
-    public static boolean NO_OBJS = false;
+    public static boolean emulateMobile;
+    public static String forcePlanetType = "";
+    public static String forceSystemType = "";
+    public static boolean noObjects;
+    static String spawnPlace = "";
 
     // Presentation
-    public static boolean NO_DRAS = false;
-    public static float ZOOM_OVERRIDE = 0;
-    public static float GRID_SZ = 0;
-    public static float GAME_SPEED_MULTIPLIER = 1f;
-    public static boolean DETAILED_MAP = false;
-    public static boolean NO_SOUND = false;
-    public static boolean SOUND_IN_SPACE = false;
-    public static boolean SHOW_WARNINGS = false;
-    public static boolean SHOW_FPS = false;
-    public static boolean DRAW_OBJ_BORDERS = false;
-    public static boolean DRAW_DRA_BORDERS = false;
-    public static boolean DRAW_PHYSIC_BORDERS = false;
-    public static boolean DRAW_PLANET_BORDERS = false;
-    public static boolean MISC_INFO = false;
-    public static boolean OBJ_INFO = false;
-    public static boolean SOUND_INFO = false;
-    public static boolean TEX_INFO = false;
+    public static boolean noDras;
```

Comment:
Let's give some of these better names please.
For example `noDrawables` should replace `noDras`

Reply:
I was wondering what does the `Dras` mean... Thanks for telling me :smile: